slurmci-test.toml

```diff
@@ -57,10 +57,12 @@ gpu = [
   { file = "test/Numerics/DGMethods/advection_diffusion/periodic_3D_hyperdiffusion.jl", slurmargs = ["--ntasks=3"], args = ["--integration-testing"] },
   { file = "tutorials/Atmos/dry_rayleigh_benard.jl", slurmargs = ["--ntasks=1"], args = ["--fix-rng-seed"] },
   { file = "tutorials/Atmos/risingbubble.jl", slurmargs = ["--ntasks=3"], args = [] },
-  { file = "experiments/AtmosGCM/heldsuarez.jl", slurmargs = ["--ntasks=1"], args = ["--diagnostics=default", "--fix-rng-seed"] },
+  { file = "experiments/AtmosGCM/heldsuarez.jl", slurmargs = ["--ntasks=1"], args = ["--diagnostics=never", "--fix-rng-seed"] },
+  { file = "experiments/AtmosGCM/baroclinic-wave.jl", slurmargs = ["--ntasks=1"], args = ["--diagnostics=never"] },
```

Comment:
Don't need `--diagnostics=never`, that's the default (unless you're calling `ClimateMachine.init(diagnostics = <interval>)`).